src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+const INITIAL_STATE = 0;
+const bookReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case ADD:
+      return [...state, action.book];
+
+    case REMOVE:
+      return state.filter((book) => book.id !== action.book.id);
+    default:
+      return state;
+  }
+};
+export const addBook = (book) => (
+  { type: ADD, book }
+);
+
+export const removeBook = (bookId) => (
+  { type: REMOVE, bookId }
+);
```

Comment:
- [ ] [OPTIONAL] Kindly use `createSlice` from `@reduxjs/toolkit` to create reducers instead of creating them manually.

Reply:
Thank you for suggesting the use of createSlice from @reduxjs/toolkit to create reducers. I will consider incorporating it into my codebase. Appreciate your feedback. 😁